migrations/20220531232510_create_oddjobs.sql

```diff
@@ -0,0 +1,116 @@
+--
+-- Name: notify_job_monitor_for_oddjobs(); Type: FUNCTION; Schema: public; Owner: postgres
+--
+
+CREATE FUNCTION public.notify_job_monitor_for_oddjobs() RETURNS trigger
+    LANGUAGE plpgsql
+    AS $$begin
+  perform pg_notify('"jobs_created_oddjobs"',
+    json_build_object('id', new.id, 'run_at', new.run_at, 'locked_at', new.locked_at)::text);
+  return new;
+end;
+$$;
+
+ALTER FUNCTION public.notify_job_monitor_for_oddjobs() OWNER TO postgres;
+
+--
+-- Name: oddjobs; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public.oddjobs (
+    id integer NOT NULL,
+    created_at timestamp with time zone DEFAULT now() NOT NULL,
+    updated_at timestamp with time zone DEFAULT now() NOT NULL,
+    run_at timestamp with time zone DEFAULT now() NOT NULL,
+    status text NOT NULL,
+    payload jsonb NOT NULL,
+    last_error jsonb,
+    attempts integer DEFAULT 0 NOT NULL,
+    locked_at timestamp with time zone,
+    locked_by text,
+    CONSTRAINT incorrect_locking_info CHECK ((((status <> 'locked'::text) AND (locked_at IS NULL) AND (locked_by IS NULL)) OR ((status = 'locked'::text) AND (locked_at IS NOT NULL) AND (locked_by IS NOT NULL))))
```

Comment:
Could this be split on multiple line? Ideally 80 chars lines but 100 is acceptable.

Reply:
yes, I did this.